tools/model_converters/swin2mmseg.py

```diff
@@ -0,0 +1,82 @@
+import argparse
+from collections import OrderedDict
+
+import torch
+from mmcv.runner import _load_checkpoint
+
+
+def swin_convert(ckpt):
```

Comment:
```suggestion
def convert_swin(ckpt):
```
It's better to use verbs + nouns.